src/components/userProfile/UserProfile.js

```diff
@@ -305,7 +306,19 @@ class UserProfile extends Component {
             tabBarTextStyle={styles.tabBarText}
             tabBarInactiveTextColor={'#a6aeae'}
             tabBarActiveTextColor={'#2e3031'}
-            tabBarUnderlineStyle={styles.tabBarUnderline}>
+            tabBarUnderlineStyle={styles.tabBarUnderline}
+            renderTabBar={() => (
+              <DefaultTabBar
+                style={
+                  {
+                    marginLeft: dimensions.widthWeight * 50,
+                    marginRight: dimensions.widthWeight * 50,
+                  }
+                }
+                containerWidth={WIDTH - dimensions.widthWeight * 100}
+                leftOffset={dimensions.widthWeight * 22}
+                rightOffset={dimensions.widthWeight * 28}
+                />)}>
             <UserOverview tabLabel='OVERVIEW' id={this.state.id}
               toggleAbout={this.toggleAbout.bind(this)}/>
             <UserCareer tabLabel='CAREER' id={this.state.id}/>
```

Comment:
34: Unexpected usage of singlequote
61: A space is required before closing bracket